src/Chainweb/Version.hs

```diff
@@ -778,13 +784,9 @@ coinV2Upgrade _ _ _ = False
 --
 twentyChainUpgrade
     :: ChainwebVersion
-    -> ChainId
     -> BlockHeight
     -> Bool
-twentyChainUpgrade Mainnet01 _ h = h == maxBound -- TODO: update me with a calculated height
-twentyChainUpgrade Development _ h = h == 150
-twentyChainUpgrade _ _ 2 = True
-twentyChainUpgrade _ _ _ = False
+twentyChainUpgrade v h = h == twentyChainUpgradeBlockHeight v
```

Comment:
This performs the update at the genesis block. Is that correct, which is hard-coded. Before it was done at a non-genesis block, actually 100 blocks after genesis.

Reply:
After reading the Haddock comment for this function, it seems that this doesn't depend on the twenty chain transition block height at all. It should trigger early after chainweb-node 1.21 is release on testnet and mainnet and at height 2 for all other versions.